src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java

```diff
@@ -87,10 +88,11 @@ public MessageWall open(String userName) {
   }
 
   public MiniEditorComponentObject triggerMessageArea() {
+    jsActions.scrollToElement(messageTitleField);
     while (!postButton.isDisplayed()) {
       jsActions.focus(messageMainBody);
     }
-    wait.forElementPresent(By.cssSelector("#wall-new-message.focused"));
+    wait.forAttributeToContain(newWallMessageContainer, "class", "focused");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "class" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)